eq-author/src/components/preview/Answers/DateAnswer.js

```diff
@@ -21,16 +21,16 @@ const DateField = styled.div`
 `;
 
 const DayDateField = styled(DateField)`
-  max-width: 6em;
+  max-width: ${props => props.enableOn ? '5em' : '6em'};
 `;
 
 const MonthDateField = styled(DateField)`
   flex: 2;
-  max-width: 15em;
+  max-width: ${props => props.enableOn ? '5em' : '15em'};
 `;
 
 const YearDateField = styled(DateField)`
-  max-width: 6em;
+  max-width: ${props => props.enableOn ? '6em' : '6em'};
```

Comment:
```suggestion
  max-width: 6em;
```

Reply:
Ah didn't catch that. Great spot. Made the change.